docs/src/main/sphinx/optimizer/statistics.md

```diff
@@ -27,6 +29,17 @@ being used and can also vary by table. For example, the
 Hive connector does not currently provide statistics on data size.
 
 Table statistics can be displayed via the Trino SQL interface using the
-{doc}`/sql/show-stats` command. For the Hive connector, refer to the
-{ref}`Hive connector <hive-analyze>` documentation to learn how to update table
-statistics.
+[](/sql/show-stats) command.
+
+Depending on the connector support, table statistics are updated by Trino when
+executing [data management statements](sql-data-management) like `INSERT`,
+`UPDATE`, or `DELETE`. For example, the [Delta Lake
+connector](delta-lake-table-statistics), the [Hive connector](hive-analyze), and
+the [Iceberg connector](iceberg-table-statistics) all support table statistics
+management from Trino. 
+
+You can also force statistics collectionmanually with the [](/sql/analyze)
```

Comment:
There's nothing forceful about it.
```suggestion
You can also make Trino collect statistics with the [](/sql/analyze)
```

Reply:
"make" is kinda bad.. I will use "initialize"